packages/msw-addon/tsconfig.browser.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@build-time/initialize": ["./src/initialize.browser.ts"]
```

Comment:
The idea of environment-specific TypeScript configs is really simple:

- Each build in `tsup.config.ts` points to a different `tsconfig.<env>.json`.
- Each different `tsconfig.<env>.json` resolves the `@build-time/initialize` import differently on _type-level_ (that's why we provide a standalone import alias in `esbuildOptions` for build-level module resolution too.

This way each compiled `.d.ts` file comes with the right types and `msw/*` imports.